website/static/js/wikiTreebeard.js

```diff
@@ -0,0 +1,117 @@
+'use strict';
+var Fangorn = require('js/fangorn');
+var $ = require('jquery');
+var bootbox = require('bootbox');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+require('../css/fangorn.css');
+
+
+function resolveToggle(item) {
+    var toggleMinus = m('i.fa.fa-minus', ' '),
+        togglePlus = m('i.fa.fa-plus', ' ');
+
+    if (item.children.length > 0) {
+        if (item.open) {
+            return toggleMinus;
+        }
+        return togglePlus;
+    }
+    item.open = true;
+    return '';
+}
+
+function resolveIcon(item) {
+    if (item.children.length > 0) {
+        if (item.open) {
+            return m('i.fa.fa-folder-open', ' ');
+        }
+        return m('i.fa.fa-folder', ' ');
+    }
+}
+
```

Comment:
I'm not sure why you need these functions resolveToggle(), resolveIcon() separately here too. Is the regular fangorn version not working for these? I'm not seeing that something specific is happening here, is that the case?